invenio_github/templates/semantic-ui/invenio_github/settings/view.html

```diff
@@ -16,10 +16,6 @@
   {%- set github_rel_url = 'https://github.com/{0}/releases/new'.format(repo.name) %}
   {% set active = true %}
 
-  {%- if latest_published %}
```

Comment:
This is [not used anywhere](https://github.com/search?q=org%3Ainveniosoftware%20%22latest_published%22&type=code) ... in fact `latest_published` is not even a view arg